feature-libs/cart/import-export/core/config/default-import-export-config.ts

```diff
@@ -21,6 +21,10 @@ export const defaultImportExportConfig: ImportExportConfig = {
           value: 'totalPrice.formattedValue',
         },
       ],
+      messageEnabled: true,
+      messageTimeout: 6000,
+      downloadDelay: 1000,
```

Comment:
why 1s? I thought that delay in web pages while downloading stuff is only because of request delay, not specified config for UX 😆

Reply:
It's for better user experience to avoid running two actions same time which may confuse user. So in this case message will show up first and after one seconds download process starts.